alcs-api/src/common/authorization/authorization.service.ts

```diff
@@ -86,4 +108,34 @@ export class AuthorizationService {
 
     return res.data;
   }
+
+  // FIXME: this will be replaced with automapper
+  private mapUserFromTokenToCreateDto(
+    user: UserFromToken,
+  ): CreateOrUpdateUserDto {
+    return {
+      email: user.email,
+      name: user.name,
+      displayName: user.display_name,
+      identityProvider: user.identity_provider,
+      preferredUsername: user.preferred_username,
+      givenName: user.given_name,
+      familyName: user.family_name,
+      idirUserGuid: user.idir_user_guid,
+      idirUserName: user.idir_username,
+    } as CreateOrUpdateUserDto;
+  }
+
+  private async registerUser(payload: Buffer) {
+    const decodedToken = JSON.parse(
+      Buffer.from(payload).toString(),
+    ) as UserFromToken;
+    this.logger.debug(decodedToken);
```

Comment:
Do you want to leave this debug in?

Reply:
yep, I assume we will have 'error' or 'warning' level on deployed env so this should not cause any issues

Reply:
Sounds good!